packages/common-scripts/src/deploy.ts

```diff
@@ -48,6 +52,7 @@ function updateCellDeps(txSkeleton: TransactionSkeletonType): TransactionSkeleto
 async function completeTx(
   txSkeleton: TransactionSkeletonType,
   fromAddress: string,
+  config?: Config,
   feeRate = BigInt(10000),
```

Comment:
should not hardcode `fee_rate` here

Reply:
add `feeRate` to `DeployOptions` ?

Reply:
I think it is fine to add `feeRate` to `DeployOptions`